include/samplog/PluginLogger.h

```diff
@@ -72,6 +72,27 @@ namespace samplog
 		{
 			return samplog::LogNativeCall(m_Module.c_str(), amx, params, name, params_format);
 		}
+
+		inline bool operator()(const char *msg)
+		{
+			return Log(LogLevel::INFO, msg);
```

Comment:
I removed this.  Personally I like it, but as you said, I can just use a derived class.